mashup.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+from __future__ import unicode_literals
+import json
+import pprint
+import sys
+import argparse
+
+
+def key_factory(field):
+    """Returns a get_key for a field"""
+    def get_key(item):
+        return item['properties'][field]
+    return get_key
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('sort', help="specify sort field. Legal fields: address, averagescore, businessname, highscore, totalinspections")
+    parser.add_argument('amount', nargs='?', type=int, help="specify number of items.")
+    parser.add_argument('direction', nargs='?', help="to reverse sort, specify reverse (must have an amount for reverse to work).")
+    args = parser.parse_args()
+
+    if args.direction == 'reverse':
+        direction = ' by reverse'
+        kargs = {'reverse': True}
+    else:
+        direction = ''
+        kargs = {}
+
+    legal_args = ['address',
+                  'averagescore',
+                  'businessname',
+                  'highscore',
+                  'totalinspections']
+
+    legal_fields = ['Address',
+                    'Average Score',
+                    'Business Name',
+                    'High Score',
+                    'Total Inspections']
+
+    if args.sort not in legal_args:
+        print 'Not a sortable field. Could not sort.'
+        sys.exit()
```

Comment:
good command-line citizens will return a non-zero exit code when an error caused the problem:

```python
sys.exit(1)
```